### PR TITLE
emscripten: update module api to 1.37.10

### PR DESCRIPTION
**Pull request by periklis**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: http://kripken.github.io/emscripten-site/docs/api_reference/module.html
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dslint/dt.json" }`.

### CODE REVIEW

File: types/emscripten/index.d.ts [hunk 1]
```diff
@@ -1,21 +1,43 @@
 // Type definitions for Emscripten
 // Project: http://kripken.github.io/emscripten-site/index.html
 // Definitions by: Kensuke Matsuzaki <https://github.com/zakki>
+//                 Periklis Tsirakidis <https://github.com/periklis>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
```
Comment (mhegazy) [inline types/emscripten/index.d.ts]: why do you need TS 2.2?

Comment (periklis) [inline types/emscripten/index.d.ts]: travis kindly pointed out, that i added `object` type. See line 33

**Comment by mhegazy** [comment]
@zakki any comments?

**Comment by zakki** [comment]
`preInit` seems duplicated.

**Comment by periklis** [comment]
@zakki  thx for the tip, i fixed that.

**Pull request merged**